packages/manager/apps/ips/src/pages/byoip/sections/OrderButtonSection.component.tsx

```diff
@@ -11,22 +11,22 @@ import { useNavigate } from 'react-router-dom';
 import { urls } from '@/routes/routes.constant';
 
 export const OrderButtonSection: React.FC = () => {
-  const { t: tcommon } = useTranslation(NAMESPACES?.ACTIONS);
+  const { t } = useTranslation([NAMESPACES?.ACTIONS]);
   const navigate = useNavigate();
 
   return (
     <div className="flex gap-3">
       <OdsButton
         color={ODS_BUTTON_COLOR.primary}
         size={ODS_BUTTON_SIZE.md}
-        label={tcommon('next')}
+        label={t('next', { ns: NAMESPACES.ACTIONS })}
         onClick={() => navigate(urls.byoipOrderModal)}
       />
       <OdsButton
         color={ODS_BUTTON_COLOR.primary}
         size={ODS_BUTTON_SIZE.md}
         variant={ODS_BUTTON_VARIANT.ghost}
-        label={tcommon('cancel')}
+        label={t('cancel', { ns: NAMESPACES.ACTIONS })}
```

Comment:
You can also write it like that fyi 

```suggestion
        label={t(`${NAMESPACES.ACTIONS}:cancel')}
```